README.md

```diff
@@ -19,11 +19,42 @@ different technologies.
 Find more information about the client configuration in the README file of the repo:
 [https://github.com/superdesk/superdesk-client-core](https://github.com/superdesk/superdesk-client-core "") 
 
-### Installation
+## Installation
+
+### Client
+
+1. Clone the repository
+2. Navigate to the folder where you've cloned this repository (if it's the main repo, go inside the `client` folder).
+2. Run `npm install && bower install` to install dependencies.
+3. Run `grunt server` to run the web server.
+
+The `grunt server` attempts to resolve the API and websockets server to a local instance. In order to use a different instance, you may add the arguments `--server=<host:[port]>` and `--ws=<host:[port]>` to the command.
+
+### Server
+
+#### Dependencies (for non-Docker installations)
+
+* Pyton & Pyvenv 3+
+* MongoDB
+* ElasticSearch
+* Redis
+
+#### MacOS
+
+For the sake of this walkthrough, it is considered that your python binary is called `python3`. Please amend as needed for your own system.
+
+1. Install dependencies using `brew` (make sure you do a `brew update` first): `python3`, `mongodb`, `elasticsearch`, `redis`.
+2. Create a virtual envirnoment in a folder of your choice, for example `mkdir ~/pyvenv && pyvenv-3.5 ~/pyvenv`.
+3. Activate the environment with by running its activation script: `. ~/pyvenv/bin/activate`. All of the following steps need to run under the virtual environment.
```

Comment:
after this you will also need to install dependencies, `pip install -r server/requirements.txt`